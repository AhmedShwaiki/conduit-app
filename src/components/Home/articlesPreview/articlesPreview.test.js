import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";
import { React } from "react";
import ArticlesPreview from "./articlesPreview";
import { BrowserRouter as Router } from "react-router-dom";

it("renders correctly without crashing", () => {
  shallow(<ArticlesPreview articles={[]} />);
});
it("renders correctly", () => {
  const tree = mount(<ArticlesPreview articles={[]} />);
  expect(toJson(tree)).toMatchSnapshot();
});
