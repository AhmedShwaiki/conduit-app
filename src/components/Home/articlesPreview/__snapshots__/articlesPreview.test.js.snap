// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ArticlesPreview
  articles={Array []}
>
  <div />
</ArticlesPreview>
`;
